cmd/notary-server/config.go

```diff
@@ -256,3 +260,34 @@ func parseServerConfig(configFilePath string, hRegister healthRegister) (context
 		ConsistentCacheControlConfig: consistentCache,
 	}, nil
 }
+
+func setupConfigReloadTrap(oldConfig *viper.Viper, configFile string, envPrefix string) {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, notary.NotarySupportedSignals...)
+	go func() {
+		for {
+			reloadConfig(<-c, configFile, oldConfig, envPrefix)
+		}
+	}()
+}
+
+func reloadConfig(sig os.Signal, configFile string, oldConfig *viper.Viper, envPrefix string) {
+	reload := func(config *viper.Viper) {
+		switch sig {
+		case syscall.SIGUSR1:
+			if err := utils.AdjustLogLevel(true); err != nil {
+				fmt.Println("Increase log level failed, will use the old one, error: ", err)
```

Comment:
Sounds good, could we also print the log level we're reverting back to?  For example: `Attempt to increase log level failed, will remain at %s level, error: ` (with the old level filling that string).  logrus has a `GetLevel` function we can use